### ISSUE TITLE
mAP extremely low when testing on Caltech

### ISSUE THREAD

**Issue by msha096** [rainofmine/Bi-box_Regression#5]
Hi,

I just found the mAP is very low when testing on Caltech Dataset, it is around 0.03
The code is 

    optimizer = optim.Adam(retinanet.parameters(), lr=1e-5)
    scheduler = optim.lr_scheduler.ReduceLROnPlateau(optimizer, patience=3, verbose=True)

If I change to SGD or I change lr=1e-4, mAP will always be 0.0 
If there anyone has similar problem as mine? What  is wrong with my expirement?

BTW, I downsize the caltech to 1/10 of the original size, it takes 35 minutes to run one epoch and I ran 50 epochs. The val data set was chosen from another 1/10 of the original Caltech Dataset. Basiclly the val and test should be very similar, but the mAP on Val is only around 0.3 at 24th epoch.

I have no idea what went wrong.
Thanks!!!

**Comment by rainofmine** [comment]
I do not test on Caltech. I wonder whether your label file is correct. Or you need to try different hyper-parameters.

**Comment by msha096** [comment]
Can I know which data set you use？ Could you please offer the code to evaluate the MR-FPPI and output the bbox？ Thanks！

**Comment by buaaswf** [comment]
@msha096 @rainofmine Thanks. Could you please share the test code?

**Comment by msha096** [comment]
> @msha096 @rainofmine Thanks. Could you please share the test code?

The test code is right in csv_eval.py which was called in train.py
